bubblesub/cmd/audio_scroll_view.py

```diff
@@ -18,6 +18,7 @@
 
 from bubblesub.api import Api
 from bubblesub.api.cmd import BaseCommand
+from bubblesub.ui.views import TargetWidget
```

Comment:
Shouldn't this be moved to the GUI API? Since `is_widget_visible` takes basically what is this enum.
I'm okay with leaving it here.

Reply:
Look two replies above, we agreed to move it out here.